smssync/src/main/java/org/addhen/smssync/presentation/util/Utility.java

```diff
@@ -278,6 +278,9 @@ public static String getPhoneNumber(@NonNull Context context, @NonNull PrefsFact
      * @return String
      */
     public static String capitalizeFirstLetter(String text) {
+        if(text == null) return null;
```

Comment:
Neat but violates coding style. Could you use the standard brace style. We use the standard android [coding style guide](https://source.android.com/source/code-style.html).